assets/docs/pages/agentgateway/llm/api-keys.md

```diff
@@ -331,12 +445,13 @@ Store the API key in a Kubernetes secret. Then, refer to the secret in the Backe
 
 ### Passthrough token {#passthrough}
 
-Pass through an existing token directly from the client or a successful OpenID Connect (OIDC) connect flow before the request is sent to the Backend. This option is useful for environments where you set up federated identity for backend clients so that they are already authenticated to the LLM providers that you create Backends for. Currently, the request must place the token in the `Authorization` header.
+Pass through an existing token directly from the client or a successful OpenID Connect (OIDC) connect flow before the request is sent to the {{% version include-if="2.1.x" %}}Backend{{% /version %}}{{% version include-if="2.2.x" %}}AgentgatewayBackend{{% /version %}}. This option is useful for environments where you set up federated identity for backend clients so that they are already authenticated to the LLM providers that you create Backends for. Currently, the request must place the token in the `Authorization` header.
```

Comment:
no worries if you think it's not worthwhile, but potentially could put this in a backend snippet, possibly even with envoy reuse

```
{{% version include-if="2.1.x" %}}Backend{{% /version %}}{{% version include-if="2.2.x" %}}AgentgatewayBackend{{% /version %}}
```

Reply:
yeah good idea. I'll do that